frontend/src/components/AddressInput.jsx

```diff
@@ -68,18 +67,18 @@ const AddressInput = () => {
         endingCoordinates: selectedCoordinates,
       }));
       setTextboxPlaceholderText("Type a starting location...");
+
+      // Pass coordinates to VehicleMap
+      onCoordinatesSelect({
+        startingCoordinates: selectedLocations.startingLocation.geometry.coordinates,
+        endingCoordinates: selectedLocations.endingLocation.geometry.coordinates,
+      });
+      
     }
 
     setLocationUpdateCount(locationUpdateCount + 1);
-    //for some reason the below line is logging 1 less than the actual count
-    console.log(
```

Comment:
i feel like we should keep this just to make sure the locationUpdate is correct bc this is what controls whether the user enters a starting or ending location